### ISSUE TITLE
Integration detail: name is missing/empty

### ISSUE THREAD

**Issue by dsimansk** [syndesisio/syndesis#430]
```html
<h1 _ngcontent-c3="">
    <syndesis-editable-text _ngcontent-c3="" _nghost-c5="">
    <!----><!---->

    <!---->
  </syndesis-editable-text>
</h1>
```

TP2
![screen shot 2017-11-23 at 08 05 50](https://user-images.githubusercontent.com/5637792/33161687-6bd6c6cc-d025-11e7-8dbd-7e90a5437326.png)

Master
![screen shot 2017-11-23 at 08 06 05](https://user-images.githubusercontent.com/5637792/33161688-6bf1b73e-d025-11e7-9529-1c349f5ac4c2.png)

**Comment by dsimansk** [comment]
@kahboom @deeleman hi guys. Could I ask you for a quick consultation. I've looked into this detail page and `syndesis-editable-text` component with intention to fix this issue. But I've found out that currently that my angular skills are far worse than required.

So currently such a code seems to be injected with any values at all.
https://github.com/syndesisio/syndesis/blob/cecfaa3b26f5c68c930d14f5b0d7b4f2103000bc/app/ui/src/app/integrations/detail-page/detail.component.html#L35-L37

But if try something like, I can eventually see the integration name and component has the value set also in HTML code. Any thoughts or advice? :)
```
<h1>
    {{ i.name }}
   <syndesis-editable-text [value]="i.name"
                                [validationFn]="validateName"
                                (onSave)="onAttributeUpdated('name', $event)"></syndesis-editable-text>
</h1>
```
